src/app/dataset-view/dataset-view-header/dataset-view-header.model.ts

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright Kamu Data, Inc. and contributors. All rights reserved.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the LICENSE file.
+ */
+
+import { SearchAdditionalHeaderButtonOptions } from "src/app/common/components/search-additional-buttons/search-additional-buttons.interface";
+import { searchAdditionalButtonsEnum } from "src/app/search/search.interface";
+
+export const searchAdditionalButtonsDescriptors: SearchAdditionalHeaderButtonOptions = {
```

Comment:
constants should be named in  UPPER_SNAKE_CASE style.

Reply:
fixed